tasks/sep/020-holocene-system-config-upgrade-multi-chain/README.md

```diff
@@ -0,0 +1,42 @@
+# Holocene Hardfork Upgrade - `SystemConfig`
+
+Status: DRAFT, NOT READY TO SIGN
```

Comment:
For sepolia, since we don't need foundation approval, we can skip directly to READY TO SIGN 
```suggestion
Status: READY TO SIGN
```

Reply:
[7e1de44](https://github.com/ethereum-optimism/superchain-ops/pull/378/commits/7e1de4404b72274448b7dbd80d4b13c63db21cf3)